resources/kubernetes/operator/helm/values.yaml

```diff
@@ -176,16 +195,17 @@ collectors:
   daemon:
```

Comment:
Because I have context, I think I can link the names cluster and daemon to be a Collector running as Deployment which collects cluster metrics and a collector running a a DaemonSet which collects node metrics, logs and routes traces.

It might not be that intutitive for users to learn that, we should add a comment that explains the blocks of configuration. 

`# Cluster Collector configuration: This Deployment-based collector gathers telemetry at the cluster level like cluster metrics.`

`# Node Collector configuration: This DaemonSet-based collector collects node-specific telemetry.`